routers/repo/view.go

```diff
@@ -103,13 +103,54 @@ func renderDirectory(ctx *context.Context, treeLink string) {
 		isTextFile := base.IsTextFile(buf)
 		ctx.Data["FileIsText"] = isTextFile
 		ctx.Data["FileName"] = readmeFile.Name()
+		fileSize := int64(0)
+		isLFSFile := false
+		ctx.Data["IsLFSFile"] = false
+
 		// FIXME: what happens when README file is an image?
 		if isTextFile {
-			if readmeFile.Size() >= setting.UI.MaxDisplayFileSize {
+			if setting.LFS.StartServer {
```

Comment:
Just use `isTextFile && setting.LFS.StartServer` in single if statement

Reply:
Sorry missed that in last minute restructuring